src/solver/solve.jl

```diff
@@ -19,14 +19,19 @@ simulated_years(es::EfficiencyStats) =
     float(es.tspan[2] - es.tspan[1]) * (1 / (365 * 24 * 3600)) #=seconds * years per second=#
 walltime_in_days(es::EfficiencyStats) = es.walltime * (1 / (24 * 3600)) #=seconds * days per second=#
 
-function timed_solve!(integrator)
+function timed_solve!(integrator, comms_ctx)
```

Comment:
Can't we obtain the `comms_ctx` from the `integrator`?

Reply:
I looked for it in the integrator but I don't think it's stored there.

Reply:
It’s certainly in there somewhere!

Reply:
This should be marked as resolved, I already updated this to get the context from the device `    comms_ctx = ClimaComms.context(device)`